### PR TITLE
New package: gopass-jsonapi-1.14.3

### PR DESCRIPTION
**Pull request by kuhnchris**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): YES

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/gopass-jsonapi/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'gopass-jsonapi'
+pkgname=gopass-jsonapi
+version=1.14.3
+revision=1
+build_style=go
+makedepends="gnupg2"
```
Comment (sgn) [inline srcpkgs/gopass-jsonapi/template:6]: `s/gnupg2/gnupg/`

Comment (kuhnchris) [reply srcpkgs/gopass-jsonapi/template]: :+1: 
I copied it from `gopass` tho, should we change it there aswell?

**Comment by kuhnchris** [comment]
fixed xlint issues.

File: srcpkgs/gopass-jsonapi/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'gopass-jsonapi'
+pkgname=gopass-jsonapi
+version=1.14.3
+revision=1
+create_wrksrc=yes
+build_style=go
+go_import_path=github.com/gopasspw/gopass-jsonapi
+makedepends="gnupg"
+depends="gnupg git gopass"
```
Comment (sgn) [inline srcpkgs/gopass-jsonapi/template:9]: `depends="gopass"` That's enough.

File: srcpkgs/gopass-jsonapi/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/gopass-jsonapi/template]: Yes, when updating `gopass`.

File: srcpkgs/gopass-jsonapi/template (hunk 2 above)
Comment (kuhnchris) [reply srcpkgs/gopass-jsonapi/template]: yessir, pushed.

**Pull request closed without merging**